EGPageViewController/Classes/EGPageViewController.swift

```diff
@@ -7,7 +7,7 @@
 import UIKit
 import SnapKit
 
-class EGPageViewController: UIPageViewController {
+public class EGPageViewController: UIPageViewController {
```

Comment:
@monolithic-adam I think this class should be subclassable, so `open` is better than `public` here. What do you think?

Reply:
@JeromeCHA You are right, I will change it to open.